src/libaktualizr/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 set(LIBAKTUALIZR_PUBLIC_HEADERS ../../include/libaktualizr/config.h
-                                ../../include/libaktualizr/types.h)
+                                ../../include/libaktualizr/types.h
+				../../include/libaktualizr/events.h
+				../../include/libaktualizr/results.h
+				../../include/libaktualizr/campaign.h) 
```

Comment:
Weird indenting change here.

Reply:
hmm, interesting, looks different in my local file